### DISCUSSION TITLE
I can't connect to the internet, too many download errors... HELP!

### DISCUSSION THREAD

**Discussion by 74IR** [BlackArch/blackarch#3926] [Support] [answered]
### Bug description
ISO I use: BlackArch Linux 64 bit Netinstall ISO 2023.04.01
Problem: The download of the iso seems to be incomplete, it doesn't let me connect to the internet, I don't have an interface terminal but if the one that comes by default, I don't have a wallpaper, I can't open any application, for example firefox or chrome.
Window manager: This happens on: All (i3-wm, fluxbox, awesome, sprectrwm)

Start date: 9 months ago, before the update.
Why not report earlier? : I didn't see it necessary at the time as I was very busy with other things but now I need it urgently, that's why I left blackarch 9 months ago when I had this problem.

**Actual result:**  It won't let me download anything, I can't open the applications, incomplete installation, terminal is missing.

**Expected result:** Complete and functional installation, which can open the window manager terminal, open chrome, the most important thing to be able to connect to the internet.

PS: I did this for vitualbox but it also happens to me if I install it native.

### Screenshots
Problema: 
![image](https://github.com/BlackArch/blackarch/assets/114535765/5a11f41b-2ed3-4095-a7d6-363e315aaa15)

i3wm
![image](https://github.com/BlackArch/blackarch/assets/114535765/c27937eb-cf65-4a38-9328-7a98f016f799)

Fluxbox:
![image](https://github.com/BlackArch/blackarch/assets/114535765/6da7958a-7bf3-4352-98c4-6626262b5bc2)

### Info for developers

**Device:** 
![image](https://github.com/BlackArch/blackarch/assets/114535765/62fdb4af-a318-4722-8ff3-fae2d61e9d8f)

**Comment by noraj** [comment]
Not many of us speak Spanish, so if you could translate your description in English please.

**Comment by ikstream** [comment]
Can you please use the exact name of the iso you used?
Also you are either not connected to a network or can't reach your DNS server.

Also you can switch to a tty, login and install something like xterm.

**Reply by 74IR** [reply]
Yes of course I did it by tty:
![image](https://github.com/BlackArch/blackarch/assets/114535765/dc32f9eb-af83-4e26-9c53-e73b1f87d750)

That's right friend, it won't let me communicate with my DNS server, the worst thing is that I can't download anything to fix it, and I haven't found a specific solution on the internet, I've tried almost everything and nothing works for me, I've already tried to change the pacman .conf the mirrorlist almost everything and nothing, the time zone too.
That's why I turn to you, since you have more advanced knowledge and I hope you can help me please.

**Comment by ikstream** [comment, marked as answer]
Have a look at [systemd-networkd](https://wiki.archlinux.org/title/systemd-networkd). Set it accordingly to your connection and you should be able to send requests out on most networks

**Reply by 74IR** [reply]
Thanks, now you can solve the problem, it happens that you just have to start the service of systemd-networkd.service
Enable it with the command systemctl start systemd-networkd.service

Thank you very much, I don't know how I couldn't decipher something so basic, the good thing is that I was able to solve it and now I have internet, THANKS
